<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LuoTunnusDialogi</class>
 <widget class="QDialog" name="LuoTunnusDialogi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uusi käyttäjätunnus</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Luomalla käyttäjätunnuksen aloitat Kitsaan pilvipalveluiden maksuttoman kokeilujakson. Saat käyttöösi Kitsaan kaikki ominaisuudet ja voit tallettaa yhden kirjanpidon Kitsaan suomalaiseen pilveen. Toki voit edelleet tallentaa kirjanpitoja myös omalle tietokoneellesi.&lt;/p&gt;&lt;p&gt;Ellet kokeilujakson aikana tee maksullista tilausta, jatkat kokeilujakson jälkeen ilmaiskäyttäjänä. Tällöin myös kokeilujakson aikana pilveen mahdollisesti luomasi kirjanpito poistetaan. &lt;/p&gt;&lt;p&gt;Ennen aloittamista tutustu Kitsaan käyttöehtoihin osoitteessa &lt;a href=&quot;https://kitsas.fi/ehdot&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;kitsas.fi/ehdot&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="ehdotCheck">
         <property name="text">
          <string>Hyväksyn pilvipalvelun käyttöehdot</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>9</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="ehtoBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Olet luomassa käyttäjätunnusta Kitsaan pilveen.&lt;/p&gt;&lt;p&gt;Aloita antamalla sähköpostiosoitteesi. Hetken kuluttua saat sähköpostiisi linkin, jonka kautta pääset viimeistelemään tunnuksesi sekä asettamaan salasanan.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Nimi</string>
           </property>
           <property name="buddy">
            <cstring>nimiEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nimiEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Sähköpostiosoite</string>
           </property>
           <property name="buddy">
            <cstring>osoiteEdit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="osoiteEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Otan yhteyttä Kitsaan palvelimelle...</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="onniViesti">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox_2">
         <property name="standardButtons">
          <set>QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LuoTunnusDialogi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LuoTunnusDialogi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>accepted()</signal>
   <receiver>LuoTunnusDialogi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ehtoBox</sender>
   <signal>rejected()</signal>
   <receiver>LuoTunnusDialogi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>582</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
